todocore/src/main/resources/application-prod.yml

```diff
@@ -13,7 +13,7 @@ spring:
     url: jdbc:postgresql://awseb-e-6kdfjg4g2j-stack-awsebrdsdatabase-8zxnwioh7p2j.cfetsdvsvkiq.eu-central-1.rds.amazonaws.com:5432/ebdb
   jpa:
     hibernate:
-      ddl-auto: validate
+      ddl-auto: create
```

Comment:
it shouldn't be create, leave it as validate. Check before doing a commit and push a check what are you changing and if there is not something necessary to change don't change it